e2e-test/app_helper.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import os
+import json
+import random
+import string
+import uuid
+import time
+
+
+# CREATE APP IN WORKSPACE
+def create_app(workspace_id, leo_url, app_type, access_scope, azure_token):
+    print(f"\nCreating {app_type} in workspace {workspace_id}...")
+    uri = f"{leo_url}/api/apps/v2/{workspace_id}/terra-app-{str(uuid.uuid4())}"
+    body = {
+        "appType": f"{app_type}",
+        "accessScope": f"{access_scope}"
+    }
+    headers = {
+        "Authorization": f"Bearer {azure_token}",
+        "accept": "application/json"
+    }
+
+    response = requests.post(url=uri, json=body, headers=headers)
+    # will return 202 or error
```

Comment:
Should we check for 202 explicitly and exit if error code is received?

Reply:
Hmm good point. I can add that assertion 👍